lib/vanagon/project.rb

```diff
@@ -665,6 +669,27 @@ def save_manifest_json
       end
     end
 
+    # Writes a yaml file at `output/<name>-<version>.<platform>.settings.yaml`
+    # containing settings used to build the current project on the platform
+    # provided (and a corresponding sha1sum file) if `yaml_settings` has been
+    # set in the projcet definition.
+    #
+    # @param [Vanagon::Platform] the platform to publish settings for
+    def publish_yaml_settings(platform)
+      return unless @yaml_settings
```

Comment:
In (most) new code in vanagon we use the attr_accessors when available, so this should be `return unless yaml_settings`